src/routes.rs

```diff
@@ -105,6 +106,15 @@ fn get_homepage<'a>() -> HTML<'a> {
     HTML(ROOT_RESPONSE)
 }
 
+/*
+* Trow health endpoint
+* GET /healthz
+ */
+#[get("/healthz")]
+fn get_healthz() -> JsonValue {
+    json!({ "status": "ok" })
```

Comment:
Just a comment on this Health endpoint, is there not more that can be tested? I.e that trow can write to the directory it needs to in order to store images?

Reply:
I know first version needs to be simple, so just a thought

Reply:
I'm actually pretty happy with this as a liveness check - it verifies that the web server is up and responding. 

We should probably check that the backend gRPC server can be reached as well, but I'm also happy to leave that for a later PR.

I guess the write test should be in a readiness check?

Reply:
Let me check on this